docker/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I don't think we need to include the license again. This should be covered by the top-level LICENSE file.